test/genall5.c

```diff
@@ -1067,7 +1067,7 @@ os_grp_0(hid_t fid, const char *group_name)
 {
     hid_t        gid  = -1;
     hid_t        fapl = -1;
-    H5F_libver_t low, high;
+    H5F_libver_t low = 0, high = 0;
```

Comment:
Rather than initializing these to 0, I feel like using `H5F_LIBVER_EARLIEST` or `H5F_LIBVER_ERROR` here would be better.

Reply:
Done with `H5F_LIBVER_ERROR`